.ci/constants.py

```diff
@@ -1,4 +1,4 @@
-BROKEN_RECIPES = set()
+BROKEN_RECIPES = set(["netifaces"])
```

Comment:
I had to disable `netifaces` cause it's failing to build due to "You need to add code for your platform."
Could be a package issue?